pkg/dns/serviceresolver.go

```diff
@@ -60,11 +64,11 @@ func NewServiceResolver(config *server.Config, accessor ServiceAccessor, endpoin
 //   * endpoint_id is "portal" when portalIP is set
 // * endpoints always returns each individual endpoint as A records
```

Comment:
update godoc with changes to returned records and description of pod dns name and returned records